src/main/java/org/prebid/server/proto/response/AmpResponse.java

```diff
@@ -10,7 +10,7 @@
 @Value
 public class AmpResponse {
 
-    Map<String, String> targeting;
+    Map<String, Object> targeting;
```

Comment:
When referring to a field in JSON document which type is unknown or arbitrary you may find that across the codebase one of Jackson's `JsonNode` or `ObjectNode` is used. Maybe we can adhere to this style and use `JsonNode` here as well (same applies to `AmpEvent`)?